### PR TITLE
Replace QuickMenuNewButton to QuickMenuMiddleButton

### PR DESCRIPTION
**Pull request by codeimpl**
QuickMenuNewButton has been implemented, and this patch rename it
because it is easier to understand the layout by renaming it to
QuickMenuMiddleButton.

Refs: #983

### CODE REVIEW

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 1]
```diff
@@ -63,8 +61,8 @@ private void updateButtons(@NonNull List<ButtonInfo> buttons, @NonNull View.OnCl
         for (final ButtonInfo info : buttons) {
             final QuickMenuButton button = new QuickMenuButton(getContext());
             button.setId(info.id);
-            button.setImageResource(info.image);
-            button.setLabel(info.label);
+            button.setCompoundDrawablesWithIntrinsicBounds(0, info.image, 0, 0);
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:64]: Add a `setIcon()` method and then use it.

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 2]
```diff
@@ -63,8 +61,8 @@ private void updateButtons(@NonNull List<ButtonInfo> buttons, @NonNull View.OnCl
         for (final ButtonInfo info : buttons) {
             final QuickMenuButton button = new QuickMenuButton(getContext());
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:62]: QuickMenuNewButton?

File: BUILD.gn [hunk 3]
```diff
@@ -198,7 +198,6 @@ android_library("triple_banana_java") {
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuDialog.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactory.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java",
-    "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuNewButton.java",
```
Comment (romandev) [inline BUILD.gn:201]: QuickMenuIconButton? or better name?

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java (hunk 1 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java]: done.

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java (hunk 2 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java]: As we discussed, QuickMenuMiddleButton is better.

File: BUILD.gn (hunk 3 above)
Comment (codeimpl) [reply BUILD.gn:201]: As we discussed, QuickMenuMiddleButton is better.

File: BUILD.gn [hunk 4]
```diff
@@ -198,7 +198,7 @@ android_library("triple_banana_java") {
...
```
Comment (romandev) [inline BUILD.gn:201]: You should have remove this file actually.

**Review by romandev — COMMENTED** [review]
Please update your commit description as per your actual change.
The commit description should be updated in commit side and then PR description also should be updated manually.

File: BUILD.gn (hunk 4 above)
Comment (codeimpl) [reply BUILD.gn:201]: sure~ thanks!

**Comment by codeimpl** [comment]
> Please update your commit description as per your actual change.
> The commit description should be updated in commit side and then PR description also should be updated manually.

Sure, I updated commit description, but PR description doesn't. So I updated both of them. Thanks.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**